set.mm

```diff
@@ -79592,21 +79592,544 @@ currently used conventions for such cases (see ~ cbvmpox , ~ ovmpox and
     ( wcel cund cfv cuni cpw c0 undefval wne pwne0 a1i eqnetrd ) ABCZADEAFZGZHA
     BIPHJNOKLM $.
 
-
 $(
 =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
   Well-founded recursion
 =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
 $)
 
+
+  $( Declare new constant symbols. $)
+  $c frecs $.
+
+  $( Declare the syntax for the well-founded recursion generator.  See
+     ~ df-frecs . $)
+  cfrecs $a class frecs ( R , A , F ) $.
+
+  ${
+    $d R f x y $.  $d A f x y $.  $d F f x y $.
+    $( This is the definition for the well-founded recursion generator.
+       Similar to ~ df-wrecs and ~ df-recs , it is a direct definition form of
+       normally recursive relationships.  Unlike the former two definitions, it
+       only requires a founded set-like relationship for its properties, not a
+       well-founded relationship.  When this relationship is also a partial
+       ordering, the proof does not use the axiom of infinity, but it requires
+       the axioms when the order is not partial.  We develop the theorems
```

Comment:
Perhaps it would be simpler to state "The proof requires either a partial order or infinity" or something, like the comment of ~ frrlem9 (this pr). This is optional